frame/support/src/dispatch.rs

```diff
@@ -2173,6 +2173,7 @@ macro_rules! impl_outer_dispatch {
 			)*
 		}
 	) => {
+		$($module::__is_call_part_defined!();)*
```

Comment:
This really shouldn't be here since this file's part of FRAMEv1. I don't think FRAMEv1 macros can actually support this, so I'll have to migrate this logic to FRAMEv2.

Reply:
I guess as long as we support FRAMEv1 then the FRAMEv1 macro should at least create a dummy `__is_call_part_defined` in order to be forward compatible

Reply:
For the case of the `Call` part, this is relatively easy since `decl_module` is mandatory (I think?) for every pallet, but for parts that aren't, they would not be able to create a dummy macro for forwards compatibility.

I think what we can do instead is since `decl_module` is mandatory, we just generate all necessary dummy macro definitions there.